project/src/views/new-evaluation/dropDownSelector.js

```diff
@@ -0,0 +1,25 @@
+const DropdownSelector = function(options = {}) {
+  
+  const arr = [`<legend class="field-title field-title-skin">${options.legends}</legend>`];
+  const constant = options.items.map((el) => {
+    const dropDownOptions= el.options.map((e) => `<option value="${e}">${e}</option>`)
+    const dropdownEl = dropDownOptions.join("");
+    const aux = el.label.split(' ').join('_');
+    
+    return `        
+            <ul>
+             <li>
+                <label class="firstLi" for="${aux}">${el.label}</label>
+                <select name="classEvalOptions" id="${aux}">
```

Comment:
Interesting `name` for this `select` box. Can you explain what it represents?

Reply:
it represents the name of my drop-down list. it isn't such a suitable name, indeed. I'll change it